index.html

```diff
@@ -80,10 +162,22 @@
 <body>
 	<div id="bg"></div>
 	<div id="info">
-		88 mph | <span id="clock">--:-- --</span> | 93&deg;F | 53%
+		<span id="clock">--:-- --</span> 53%
 	</div>
-	<div id="camera-container">
-		<video id="camera"></video>
+	<video id="camera"></video>
+	<div id="speedometer-container">
+		<canvas id="speedometer"></canvas>
+		<div id="speed"></div>
 	</div>
+	<table id="power-info">
+		<tr>
+			<td class="label">Battery voltage</td>
+			<td><span class="sensor">12.60 V</span></td>
```

Comment:
I don't really see a reason for this being on the driver's display - it's just a more raw version of the battery percentage, and I don't think the driver would ever need to know this.

Reply:
The specific choices of sensors here were mostly just placeholders to test the layout. In the Driver HUD documentation it has "voltage?" listed as a possible thing for the driver to see, and I assumed that meant battery voltage.

Reply:
Ahh indeed it does. If the values are just placeholders then it's fine, but I don't think that actually needs to be included in the final HUD.